src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int bothside;
+    private int height;
+
+    public RightTriangle(String color, int height, int bothside) {
+        super(color);
+        this.height = height;
+        this.bothside = bothside;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: Right Triangle");
```

Comment:
Use only English in messages/code. Replace 'Figure: Right Triangle' with 'Figure: right triangle' to maintain consistency with English capitalization rules.